schemas/user.py

```
@@ -0,0 +1,37 @@
+from pydantic import BaseModel, EmailStr
+from typing import Optional, List
```

Ruff (0.14.13)

`typing.List` is deprecated, use `list` instead

(UP035)